### DISCUSSION TITLE
Alert on my github Page

### DISCUSSION THREAD

**Discussion by Luuan11** [community/community#39755] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hii,

why my github.io has a alert of pishing ? 
![image](https://user-images.githubusercontent.com/79935555/203330886-a98ea5a0-66ad-41ba-bcd8-d8ebfcca257b.png)

Check the url: https://transparencyreport.google.com/safe-browsing/search?url=https:%2F%2Fluuan11.github.io%2Finstagram-login-page&hl=en-US

Could someone help me?

**Comment by airtower-luna** [comment]
Guessing from the URL I suppose you made a clone of the Instagram login page, which then got reported for phishing to the Google Safe Browsing blocklist. Check [their documentation](https://developers.google.com/search/docs/monitor-debug/security/social-engineering), there's a section about fixing issues. It'll involve removing the clone, and requesting a review after.

I've seen a lot of people with issues like this around the forum in the last few days. I wonder if someone is ignorantly handing out "make a clone of […]" as an assignment? :thinking:

**Comment by Luuan11** [comment, marked as answer]
basics, after I remove the deploy, the alert has been removed !
